lib/trino-parquet/src/main/java/io/trino/parquet/reader/Int64TimestampMillisColumnReader.java

```diff
@@ -52,6 +53,9 @@ else if (type instanceof TimestampType) {
                     type.writeObject(blockBuilder, new LongTimestamp(epochMicros, 0));
                 }
             }
+            else if (type == BIGINT) {
+                type.writeLong(blockBuilder, utcMillis);
```

Comment:
Maybe the var should be renamed to `epochMillis` in a separate commit too?

Reply:
Yes, that seems like a good idea. Do you want a separate PR?

Reply:
A separate commit in this PR should be fine.

Reply:
Alright, have done so